package.json

```diff
@@ -23,6 +23,7 @@
     "trailingComma": "none"
   },
   "dependencies": {
+    "@dcl-sdk/utils": "^1.2.8",
```

Comment:
I'll try to avoid adding this lib here. But for now lets continue this way and then we can remove it.